stable/redis/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: redis
-version: 9.2.2
+version: 9.2.3
```

Comment:
Please bump the minor version as you are adding a new feature.

Reply:
Done! Had to force push because I forgot to sign a couple of my commits and had to alter the commit messages.